sphinx/source/conf.py

```diff
@@ -179,10 +180,44 @@
 breathe_projects = {"CCF": "../../doxygen/xml"}
 breathe_default_project = "CCF"
 
+"""
 html_context = {
     "source_url_prefix": "https://github.com/Microsoft/CCF",
     "display_github": True,
     "github_user": "Microsoft",
     "github_repo": "CCF",
     "github_version": "master/sphinx/source/",
 }
+"""
+
+# Set up multiversion extension
+
+smv_tag_whitelist = r"^v.*$"
+smv_branch_whitelist = r"^master$"
+smv_remote_whitelist = None
+smv_released_pattern = r"^tags/.*$"
```

Comment:
Is this the only thing choosing which versions we build? If so, we may want to make this a semver regex, so we don't render temporary/debugging tags.

Reply:
No, this is just some weird categorisation I don't think we need: https://holzhaus.github.io/sphinx-multiversion/master/templates.html#version-listings

What chooses what we build is the tag whitelist, and the branch whitelist. Then we can split them visually, but that didn't seem all that necessary. I guess I might as well remove this line to avoid confusion.

Reply:
Ah right, cool!